resources/src/resources/topicHeader.py

```diff
@@ -24,8 +24,8 @@
 CONTROL_SEABOTIX="/kraken/control/seabotixData"
 CONTROL_LOADPARAM="/kraken/control/loadparam_srv"
 SENSOR_IMU="/kraken/sensor/imu"
-SENSOR_IMU_NEW="/kraken/sensor/imu_new"
```

Comment:
There is no need to change that order. Also, you should add this new variable at the end of the `SENSOR_` namespace.